nx.json

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": "nx/presets/core.json",
+  "npmScope": "remix",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build-all",
+          "validate-all",
+          "build:css",
+          "build:remix",
+          "build:server",
+          "typecheck",
+          "test:run",
+          "lint",
+          "test:e2e:run"
+        ],
+        "accessToken": "ZTI3Y2Q2YTUtZTM0ZC00MjFkLWJkZDctNDZmODIxMzhmZTM3fHJlYWQtd3JpdGU="
```

Comment:
Do people need a different `accessToken` when they generate this project? I think it would be better if there weren't an `accessToken` at all. I seem to remember you mentioning that it's not required. In fact, I think it would probably be best if we don't use nx-cloud in the template at all. Won't caching still happen locally without nx-cloud?